lib/Service/SuperService.php

```diff
@@ -16,4 +16,39 @@ public function __construct(LoggerInterface $logger, ?string $userId, Permission
 		$this->logger = $logger;
 		$this->userId = $userId;
 	}
+
+	public function AdminNotificationCall($tableId,$viewId) {
+	 
+        $url = 'http://admin:admin@localhost/ocs/v2.php/apps/notifications/api/v2/admin_notifications/admin';
```

Comment:
This is not the expected API to be used, the reference is about https://github.com/nextcloud/notifications/blob/master/docs/notification-workflow.md which is a PHP API provided by Nextcloud server. There is no need to perform an HTTP request here especially with passing username/password.